scripts/ec2cleanup.py

```diff
@@ -46,10 +46,11 @@ def delete_old_instances(texts, ec2provider, provider_key, date,
             report.write("\n{}:\n-----------------------\n".format(provider_key))
             for instance in ec2provider.list_vm(include_terminated=True):
                 creation = ec2provider.vm_creation_time(instance)
-                print("EC2:{}  {}  \t {}".format(
-                    provider_key, instance, (date - creation)))
-                report.write("EC2:{}  {}  \t {}\n".format(
-                    provider_key, instance, (date - creation)))
+                message = "EC2:{}  {}  \t {} \t {} \t {}\n".format(
```

Comment:
Consider labeling these fields, might become bothersome to parse the output/log with just a block of numbers/strings